.github/workflows/scripting.yml

```diff
@@ -19,8 +19,6 @@ name: Scripting
 on:
   push:
     branches: [ "master" ]
-    paths:
```

Comment:
Maybe have a path filter for `src/`?

Reply:
Sure, but generally all workflows should be made to have that, I believe.